vfs_appointment_bot/vfs_appointment_bot.py

```diff
@@ -33,9 +33,13 @@ def _read_command_line_args():
 
     _vfs_client = _VfsClient()
     _config_reader = _ConfigReader()
-    _interval = _config_reader.read_prop("DEFAULT", "interval");
+    _interval = _config_reader.read_prop("DEFAULT", "interval")
+    _use_telegram = eval(_config_reader.read_prop("DEFAULT", "use_telegram"))
+    _use_twilio = eval(_config_reader.read_prop("DEFAULT", "use_twilio"))
     logging.debug("Interval: {}".format(_interval))
-    
+    logging.debug("Will use Telegram : {}".format(_use_telegram))
+    logging.debug("Will use Twilio : {}".format(_use_twilio))
```

Comment:
This log is redundant. Can you please remove it? and also the `__use_telegram` and `_use_twilio` variable as we are already reading the props and logging it in VFSClient

Reply:
Good catch. Removed it in latest commit. :)